### PR TITLE
fix output/input files to PTModel and PTPredict

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
# Description

PR adds registration of output and input files of PTModel and PTPredict. #4401 

Needs to be tested once more before merging!

### CODE REVIEW

**Comment by Leon-Bichmann** [comment]
tested PTModel and PTPredict locally on my mac and they run through

File: src/topp/PTModel.cpp [hunk 1]
```diff
@@ -596,10 +600,28 @@ class TOPPPTModel :
 
     svm.saveModel(outputfile_name);
 
+
```
Comment (FabianAicheler) [inline src/topp/PTModel.cpp:603]: kA was andere style gurus zu neuen leeren Zeilen sagen :D

Comment (Leon-Bichmann) [reply src/topp/PTModel.cpp]: removed

**Comment by FabianAicheler** [comment]
Looks okay to me.

**Comment by timosachsenberg** [comment]
rebuild jenkins

**Comment by Leon-Bichmann** [comment]
jenkins worked now

**Comment by timosachsenberg** [comment]
ok nice - one step further but travis is failing. can you take a look

**Comment by Leon-Bichmann** [comment]
this time it went through

**Pull request merged**